tabulon_vello/src/lib.rs

```diff
@@ -80,7 +80,7 @@ impl Environment {
                     }) => {
                         let transform = graphics.get_transform(*transform);
 
-                        let mut builder = layout_cx.ranged_builder(font_cx, text, 1.0);
+                        let mut builder = layout_cx.ranged_builder(font_cx, text, 1.0, false);
```

Comment:
Just to be sure, we don't want the pixel snapping behavior here or below?

Reply:
Correct, we don't know what scale we are rendering these at.